src/lightly_train/_models/timm/timm_package.py

```diff
@@ -54,6 +54,10 @@ def get_model(
         args = dict(pretrained=False)
         if model_args is not None:
             args.update(model_args)
+        # vit and eva models have dynamic_img_size defaulting to False, which would not allow inputs with varying image sizes, e.g., for DINO
+        if model_name.startswith("vit") or model_name.startswith("eva"):
```

Comment:
This line has to be above the previous `if` statement. `model_args` must always overwrite anything that is in `args`, otherwise users cannot overwrite our defaults.